_integration-tests/tests/redigo/redigo.go

```diff
@@ -95,7 +63,10 @@ func (tc *TestCase) Run(t *testing.T) {
 
 	client, err := tc.Pool.GetContext(ctx)
 	require.NoError(t, err)
-	defer func() { require.NoError(t, client.Close()) }()
+	defer func() { assert.NoError(t, client.Close()) }()
```

Comment:
I'm surprised the linter doesn't complain about using `assert` to verify the absence of an error... But cool :)